### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by dendisuhubdy** [Tessil/ordered-map#14]
```
$ ./test_ordered_map
libc++abi.dylib: terminating with uncaught exception of type boost::unit_test::framework::setup_error: test unit with name 'test_insert<tsl__ordered_map<long long, long long, std____1__hash<long long>, std____1__equal_to<long long>, std____1__allocator<std____1__pair<long long, long long> >, std____1__deque<std____1__pair<long long, long long>, std____1__allocator<std____1__pair<long long, long long> > > >>' registered multiple times
[1]    48387 abort      ./test_ordered_map
```

**Comment by Tessil** [comment]
Hello,

Thank you for the report. Which compiler do you use as I can't reproduce the problem on my side (neither with GCC, Clang or MSVC)?

Nonetheless, I removed some duplicate types in `test_types` of `tests/ordered_map_tests.cpp` which could be the reason of the `registered multiple times` error. Could you try it with the last commit?

**Comment by Tessil** [comment]
With which version of Boost?

Does the last commit resolves the problem?

**Comment by dendisuhubdy** [comment]
It resolved it! I'm using `Boost version: 1.67.0`. Thank you!
